src/hardware/adlib_gold.cpp

```diff
@@ -146,7 +146,7 @@ void StereoProcessor::Reset() noexcept
 }
 
 void StereoProcessor::ControlWrite(const StereoProcessorControlReg reg,
-                                   const uint8_t data) noexcept
```

Comment:
I actually copied that style from some of your more recent additions. So basically we won't bother with adding `noexcept` to methods from now one?

Reply:
This `noexcept` removal was only needed because the ControlWrite calls into .setup() function that contains a `throw` (and throws inside noexcept result in a crash).

I think it's good to use `noexcept` when we're certain of it; I think it can help the compiler eliminate the exception handling pathways . 

C++17 has become even more supportive of developers trying to eliminate exceptions entirely, like in the std::filesystem, where they (usually) have two types of calls - on that throws and another using a return code:

https://en.cppreference.com/w/cpp/filesystem/is_directory

Reply:
Ah okay, thanks for the clarification, makes sense.